### ISSUE TITLE
Rubric

### ISSUE THREAD

**Issue by rrmcinty** [d-dtm/Kakuro-Helper#6]
The rubric shows: crc cards- 15%, user interface, cursor enters cell, type number into cell, board update, delete existing cell, game/read, game/save, contribution - 10%, game/new 5%. 

I have my program doing most of this, but its unorganized unlike yours which is very organized and put together. I don't know how to work with so many different classes/packages interacting with each other eg (using a boardHandler as a separate class or pulling cell objects from a different package to the calculator). I'm going to repost what I have on graphics version (its very long and messy) and I think we should at least get it the other working to this bare minimum. 

In short: we should get the project working to the minimum standards of the rubric (opening a game file/displaying it) and then worry about the math portion. What do you think?

**Comment by d-dtm** [comment]
That sounds great, I'll push what I have so you can see what I've been working on and we can work on merging to the master.
